flow/aggregates.py

```diff
@@ -497,7 +529,7 @@ def __getitem__(self, id):
         try:
             return (self._project.open_job(id=id),)
         except KeyError:
-            raise LookupError(f"Did not find aggregate with id {id}.")
+            raise KeyError(f"Aggregate id {id} is not in this aggregate store.")
```

Comment:
Since the store classes are private, I think we should not mention about them anywhere in the "user-space".

Reply:
There is one more place in aggregates.py where this error message is thrown.

Reply:
Seems reasonable to me. I'll revert the message but I'll keep it as a `KeyError`.